src/components/card/index.jsx

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line react/prop-types
+export default function Card({ title = '', img }) {
+  return (
+    <div className="shop1">
```

Comment:
Should be `shop` instead of `shop1`. In general we shouldn't use `numbers` in class names